docs/docs/examples/workflow/multi_step_query_engine.ipynb

```diff
@@ -0,0 +1,493 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        return "none" in query_bundle.query_str.lower()
</pre><p>what is this checking for actually? When would this be none?</p><p> </p><p>I think the answer is the step decompose query transform probably returns none if there are no more steps?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/15438/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
The updated query based on previous reasoning will be none if there are no more steps needed to answer the query and so we will stop the process.